src/huggingface_hub/inference/_common.py

```diff
@@ -147,10 +147,6 @@ def _get_recommended_model(task: str) -> str:
             f"Task {task} has no recommended task. Please specify a model explicitly. Visit"
             " https://huggingface.co/tasks for more info."
         )
-    logger.info(
```

Comment:
This had to be promoted up one level, so `get_recommended_model` wouldn't have this extra log message incorrectly show up